script/Arcueid/Day 8/08_00_ARC08_8_1B.txt

```diff
@@ -408,8 +408,8 @@ According to what I heard from my master,
 [sha:1a04df8d9c9abb4519c9499fa4c7098e0b593edf]{
 -- Page 572, Offset 21211.
 -- あんがい身近なところにいて、何十年も前から潜伏しているケースのが多いの。それも<街|・><で|・><い|・><ち|・><ば|・><ん|・><力|・><の|・><あ|・><る|・>、<権|・><力|・><者|・><の|・><家|・>にとかね」
-There have been a number of cases where they were hiding unexpectedly close by for decades.
- Often in the %{i}homes of the families who wielded the most power in the city%{/i}."
+There have been a number of cases where they were hiding%{n}unexpectedly close by for decades.
```

Comment:
this line is the most egregious if you want to use it as an example in looking at deepluna.